### PR TITLE
XW-4060 | Fix notifications on Edge

### PR DESCRIPTION
**Pull request by xkxd**
https://wikia-inc.atlassian.net/browse/XW-4060

@Wikia/x-wing

### CODE REVIEW

File: extensions/wikia/BannerNotifications/js/BannerNotifications.js [hunk 1]
```diff
@@ -112,6 +113,7 @@
 			this.timeout = defaultTimeout;
 		}
 
+		debugger;
```
Comment (WikiaTech) [inline extensions/wikia/BannerNotifications/js/BannerNotifications.js]: CRITICAL Remove this debugger statement. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ADebuggerStatement)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
* ![CRITICAL][CRITICAL] 1 critical

Watch the comments in this conversation to review them.

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'

File: extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-components/_banner-notification.scss [hunk 2]
```diff
@@ -2,6 +2,11 @@
 	background-color: $wds-color-white;
 	color: $wds-color-light-gray;
 	display: flex;
+	transition: opacity 0.4s;
+
+	&--transparent {
```
Comment (latata) [inline extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-components/_banner-notification.scss]: we do not use BEM modifiers in DS. I'd change it to `&.wds-is-transparent` or `&.wds-is-hidden`

Comment (vforge) [reply extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-components/_banner-notification.scss]: See https://github.com/Wikia/design-system/pull/169#discussion_r142486046

**Review by latata — APPROVED** [review]

File: extensions/wikia/DesignSystem/bower.json [hunk 3]
```diff
@@ -3,6 +3,6 @@
   "private": true,
 
   "dependencies": {
-    "design-system": "Wikia/design-system#8.8.0"
```
Comment (rybmat) [inline extensions/wikia/DesignSystem/bower.json:6]: remember to update to release version

**Review by rybmat — CHANGES_REQUESTED** [review]

**Review by rybmat — APPROVED** [review]

**Review by hakubo — APPROVED** [review]

**Pull request merged**